Formula/llvm.rb

```diff
@@ -381,8 +402,21 @@ def caveats
               "-I#{MacOS.sdk_path}/usr/include",
               "-L#{lib}",
               "-Wl,-rpath,#{lib}", "test.cpp", "-o", "test"
-      assert_includes MachO::Tools.dylibs("test"), "#{opt_lib}/libc++.1.dylib"
+      assert_includes MachO::Tools.dylibs("test"), "@rpath/libc++.1.dylib"
```

Comment:
No, this was a breaking change made in brew itself in order to fix `rust`. There are any number of formulae, among which is `llvm`, that are impacted.